src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/GenerateCoverSheetServiceTest.java

```diff
@@ -182,8 +183,14 @@ private void assertCoversheetCalledWithRequiredData() {
             any(), any());
         Map<String, Object> data = generateDocumentCaseDetailsCaptor.getValue().getData();
 
+        String expectedCourtContactDetails = "HMCTS Financial Remedy" + "\n"
+            + "PO BOX 12746" + "\n"
+            + "HARLOW" + "\n"
+            + "CM20 9QZ";
+
         assertThat(data, hasKey(ADDRESSEE));
-        assertThat(data, hasKey(CTSC_CONTACT_DETAILS));
+        assertThat(data, hasKey(COURT_CONTACT_DETAILS));
+        assertEquals(expectedCourtContactDetails, data.get(COURT_CONTACT_DETAILS));
```

Comment:
can we add it tests for the other operator checks we do i.e. null and empty string